packages/react/src/components/Tabs/Tabs-story.js

```diff
@@ -122,7 +157,15 @@ storiesOf('Tabs', module)
           {...props.tab()}
           label="Tab label 3"
           renderContent={TabContentRenderedOnlyWhenSelected}>
-          <div className="some-content">Content for third tab goes here.</div>
+          <div className="some-content">
+            <p>Content for third tab goes here.</p>
```

Comment:
Since you correctly added some `p` tags here, the other tabs are looking a bit off 😄 Would you mind adding `p` tags for the other tabs, just so it renders consistently when displayed on the website? Thanks!

Reply:
Thank you! 🙏